### PR TITLE
Document for API detail book

### PR DESCRIPTION
**Pull request by at-minhdao**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Product Backlog ID: PBI-38
- Task ID: #38.1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:
-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Non-SQL

### CODE REVIEW

File: resources/views/api-doc-builders/index.blade.php [hunk 1]
```diff
@@ -1,2 +1,4 @@
 {!! file_get_contents(base_path('docs/api-references/api.md')) !!}
 {!! file_get_contents(base_path('docs/api-references/sample.md')) !!}
+{!! file_get_contents(base_path('docs/api-references/book.md')) !!}
+
```
Comment (at-hieule) [inline resources/views/api-doc-builders/index.blade.php]: Dư dòng bạn ơi

File: docs/api-references/book.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+```
+/api/books/{id}
+```
+Get a detail book fail, id of book doesn't exist
+#### Request Headers
+| Key | Value |
+|---|---|
+|Accept|application\json
+
+#### Response
```
Comment (at-hieule) [inline docs/api-references/book.md]: Response nên phân ra success và fail. Không cần `request headers` lặp lại

**Review by at-hieule — COMMENTED** [review]
Please check my cmt

**Comment by at-hieule** [comment]
LGTM

File: docs/api-references/book.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+## Books API
+
+### `GET` Book
+```
+/api/books/{id}
```
Comment (at-duyho) [inline docs/api-references/book.md]: - nên mô tả thông tin của response. (tên, mô tả, kiểu dữ liệu, ví dụ)

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment

**Review by at-trampham — COMMENTED** [review]
LGTM

**Comment by at-trampham** [comment]
LGTM

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**